<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenAerialMapDialogBase</class>
 <widget class="QDialog" name="OpenAerialMapDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Aerial Map (OAM)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_widget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="imagery_tab">
      <attribute name="title">
       <string>Imagery</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>109</y>
         <width>61</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="add_source_button">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Add selected imagery source</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_source_button">
          <property name="toolTip">
           <string>Remove selected imagery source</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>104</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QPushButton" name="up_source_button">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Move selected source up</string>
          </property>
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="down_source_button">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Move selected source down</string>
          </property>
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>25</y>
         <width>246</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="reorganize_label">
          <property name="text">
           <string>Sources in order of upload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="sources_list_widget">
          <property name="toolTip">
           <string>Reorganize and change the upload order</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>25</y>
         <width>231</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="select_source_label">
              <property name="maximumSize">
               <size>
                <width>191</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>From layers</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="layers_tool_button">
              <property name="toolTip">
               <string>Reload layers</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="layers_list_widget">
            <property name="toolTip">
             <string>Select layers to be uploaded</string>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="file_label">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>From file</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QLineEdit" name="source_file_edit">
            <property name="toolTip">
             <string>Select file using tool button on the right </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="file_tool_button">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="imagery_next_button">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>265</y>
         <width>66</width>
         <height>27</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Go to the next tab</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="metadata_tab">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>30</y>
         <width>321</width>
         <height>290</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_1">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="title_label">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="title_edit">
          <property name="toolTip">
           <string>Human friendly title of the image</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="platform_label">
          <property name="text">
           <string>Platform:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="platform_combo_box">
          <item>
           <property name="text">
            <string>satellite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>aircraft</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UAV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>balloon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kite</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="sensor_label">
          <property name="text">
           <string>Sensor:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="sensor_edit">
          <property name="toolTip">
           <string>Image aquisition device (eg. camera, radar, etc)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="sense_start_label">
          <property name="text">
           <string>Sense start</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDateTimeEdit" name="sense_start_edit"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="sense_end_label">
          <property name="text">
           <string>Sense end</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDateTimeEdit" name="sense_end_edit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="tags_label">
          <property name="text">
           <string>Tags:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="tags_edit">
          <property name="toolTip">
           <string>Free tags separated by comma</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="provider_label">
          <property name="text">
           <string>Provider:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="provider_edit">
          <property name="toolTip">
           <string>Provider/owner of the OIN bucket</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="contact_label">
          <property name="text">
           <string>Contact:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="contact_edit">
          <property name="toolTip">
           <string>Name and email address of the data provider</string>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="website_label">
          <property name="text">
           <string>Website:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="website_edit">
          <property name="toolTip">
           <string>Website of the data provider</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QCheckBox" name="license_check_box">
          <property name="text">
           <string>In accordance with OAM free license terms</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>615</x>
         <y>90</y>
         <width>87</width>
         <height>95</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="default_button">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clean_button">
          <property name="text">
           <string>Clean</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>30</y>
         <width>226</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="select_source_metadata_label">
          <property name="text">
           <string>Sources to edit metadata</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="added_sources_list_widget">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>615</x>
         <y>245</y>
         <width>87</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QPushButton" name="metadata_previous_button">
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="metadata_next_button">
          <property name="toolTip">
           <string>Go to the next tab</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>formLayoutWidget_2</zorder>
      <zorder>metadata_next_button</zorder>
      <zorder>metadata_previous_button</zorder>
     </widget>
     <widget class="QWidget" name="upload_tab">
      <attribute name="title">
       <string>Upload</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>30</y>
         <width>258</width>
         <height>266</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="review_metadata_label">
          <property name="text">
           <string>Review sources metadata</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="metadata_review_browser">
          <property name="toolTip">
           <string>Review full metadata to be uploaded and go back to the Metadata tab in case changes need to be done</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>315</x>
         <y>175</y>
         <width>266</width>
         <height>109</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="storage_label">
          <property name="text">
           <string>AWS Bucket</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="storage_combo_box">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>oam-qgis-plugin-test</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>other...</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="key_id_label">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Key-id</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="key_id_edit">
          <property name="toolTip">
           <string>AWS access key id</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="secret_key_label">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Secret key</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="secret_key_edit">
          <property name="toolTip">
           <string>AWS secret access key</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="specify_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>   Specify</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="specify_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Bucket name which is not listed above </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>315</x>
         <y>55</y>
         <width>261</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="notify_oam_check">
          <property name="text">
           <string>Notify OAM of new resource</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="trigger_tiling_check">
          <property name="text">
           <string>Trigger OAM tile service</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="reproject_check">
          <property name="text">
           <string>Reproject to EPSG:3857</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="convert_geotiff_rgb_check">
          <property name="text">
           <string>Convert format to GeoTIFF RGB</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>605</x>
         <y>70</y>
         <width>91</width>
         <height>186</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="quit_button">
          <property name="toolTip">
           <string>Quit plugin</string>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upload_previous_button">
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upload_button">
          <property name="toolTip">
           <string>Upload imagery and metadata</string>
          </property>
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
